.github/fabricbot.json

```diff
@@ -996,6 +996,46 @@
         "taskName": "Approve PRs to release branches that are approved by InfraSwat"
       },
       "disabled": true
+    },
+    {
+      "taskType": "trigger",
+      "capabilityId": "IssueResponder",
+      "subCapability": "PullRequestResponder",
+      "version": "1.0",
+      "config": {
+        "conditions": {
+          "operator": "and",
+          "operands": [
+            {
+              "name": "prMatchesPattern",
```

Comment:
I think we should also filter out PRs by dotnet-bot.
We might also want to filter out PRs to feature branches (though, these days people usually just disable the analyzer which errors on PublicAPI mismatch in the feature branch.)

Reply:
Fair points.

Reply:
I don't think we can do feature branches sanely. There doesn't appear to be a regex match in fabric bot, so we'd have to list (and constantly update) the branches.